include/mbedtls/mbedtls_config.h

```diff
@@ -129,7 +129,11 @@
  * MBEDTLS_PLATFORM_TIME_MACRO, MBEDTLS_PLATFORM_TIME_TYPE_MACRO and
  * MBEDTLS_PLATFORM_STD_TIME.
  *
- * Comment if your system does not support time functions
+ * Comment if your system does not support time functions.
+ *
+ * \note Please note that, for suitable platforms and without MBEDTLS_TIMING_ALT
```

Comment:
style: we seem to not use pleasantries such as "please note that" in `\note` sections, but just go straight into the detail:

```
\note For suitable platforms and without MBEDTLS_TIMING_ALT...
```

I also wonder if we should mention `MBEDTLS_TIMING_C` specifically. Something like this, if it is correct:

```
\note If MBEDTLS_TIMING_C is set - to enable the semi-portable timing interface -
timing.c will include time.h regardless of the setting of MBEDTLS_HAVE_TIME,
unless MBEDTLS_TIMING_ALT is used.
```

And we should definitely say something in the entry for `MBEDTLS_TIMING_C`

Reply:
I already added the same description to the entry of `MBEDTLS_TIMING_C` previously, but I like your version much more.
I added it and mentioned you as a co-author. Thanks!